src/utils/Toast.tsx

```diff
@@ -152,8 +153,54 @@ function info(options: Options) {
   );
 }
 
+/**
+ * Displays toast notification with a trash icon upon deletion.
+ *
+ * @param options An object containing options for the toast notification.
+ * @param options.title The title of the toast notification.
+ * @param options.message The message to display in the toast notification.
+ * @param options.hintMessage An optional hint message to display in the toast notification.
+ * @param options.hintOnClick An optional callback function to execute when the hint message/checkbox is clicked.
+ * @param options.buttons An array of button-labels to display in the toast notification. (for e.g. ["Save", "Cancel"])
+ * @param options.firstButtonOnClick An optional callback function to execute when the first button is clicked.
+ * @param options.secondButtonOnClick An optional callback function to execute when the second button is clicked.
+ * @param options.autoClose The time in milliseconds after which the toast notification should automatically close.
+ * @param options.icon An optional icon to display in the toast notification.
+ * @param options.iconName An optional icon Name. Needs to be set to a any different name than "success", "error", or "info" when an icon is passed aswell.
+ */
+function deletion(options: Options) {
```

Comment:
It would be kinda cool to make it less specific and have a generic `custom(...)` which you can give a custom icon over the options as ReactNode